nats-server/configuration/logging.md

```diff
@@ -64,12 +64,15 @@ All of these settings are available in the configuration file as well.
 debug:   false
 trace:   true
 logtime: false
+log_size_limit: 1GB
 log_file: "/tmp/nats-server.log"
 ```
 
-### Log Rotation with logrotate
+### Log Rotation
 
-NATS server does not provide tools to manage log files, but it does include mechanisms that make log rotation simple. We can use this mechanism with [logrotate](https://github.com/logrotate/logrotate); a simple standard Linux utility to rotate logs available on most distributions like Debian, Ubuntu, RedHat \(CentOS\), etc.
+Introduced in NATS Server v2.1.4, NATS allows for auto-rotation of log files when the size is greater than the configured limit. The backup files will have the same name as the original log file with the suffix .yyyy.mm.dd.hh.mm.ss.micros.
```

Comment:
Should we hint to the name of the configuration param (`logfile_size_limit`) anywhere in this paragraph?

Reply:
added the param in the paragraph - that enough?